coil-base/src/main/java/coil/ImageLoader.kt

```diff
@@ -51,15 +63,15 @@ interface ImageLoader {
      * @param request The request to execute.
      * @return A [RequestDisposable] which can be used to cancel or check the status of the request.
      */
-    fun load(request: LoadRequest): RequestDisposable
+    fun launch(request: LoadRequest): RequestDisposable
```

Comment:
Could be similar to OkHttp and do `enqueue` but `launch` is good too.

Reply:
Good point. I mostly landed on `launch` since it's the keyword to start a new coroutine. Also feels weird to me to have `enqueue`, but no `execute`.

Reply:
Very true. Makes sense to me!